nautobot/extras/templates/extras/plugins_list.html

```diff
@@ -3,10 +3,35 @@
 {% load static %}
 
 {% block content %}
+    <div class="row noprint">
+        <div class="col-md-12">
+            <ol class="breadcrumb">
+                {% block breadcrumbs %}
+                    <li><a href="{% url 'apps:apps_list' %}">Installed Apps</a></li>
+                {% endblock breadcrumbs %}
+            </ol>
+        </div>
+    </div>
+
     <div class="pull-right noprint">
+        <div class="btn-group">
+            <a class="btn btn-default set-plugins_list-display" data-display="tiles" href="" title="Tiles">
+                <i class="mdi mdi-view-grid{% if display == "list" %}-outline{% endif %}"></i>
+            </a>
+            <a class="btn btn-default set-plugins_list-display" data-display="list" href="" title="List">
+                <i class="mdi mdi-view-sequential{% if display == "tiles" %}-outline{% endif %}"></i>
+            </a>
+        </div>
+
         {% if request.user.is_authenticated and table_config_form %}
-            {% table_config_button table table_name="ObjectTable" %}
+            {% if display == "tiles" %}
+                {% table_config_button table table_name="ObjectTable" disabled=True %}
+            {% else %}
+                {% table_config_button table table_name="ObjectTable" %}
+            {% endif %}
```

Comment:
I was banging my head against the wall how to make `disabled` a boolean rather than to duplicate the same `table_config_button` block with and without `disabled` arg

Reply:
can you do `disabled=(display == "tiles")`?

Otherwise I think this is fine.

Reply:
unfortunately not, this does not work either :cry: